packages/react/src/components/breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { DropdownDirection } from '../header/components/navigationLink';
+import { Header } from '../header/Header';
+import { Breadcrumb } from './Breadcrumb';
+
+export default {
+  component: Breadcrumb,
+  title: 'Components/Breadcrumb',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {
+    list: [
+      { title: 'Front page', path: '/' },
+      { title: 'Health and social services', path: '/path' },
+      { title: 'Senior services ', path: '/path/2ndLevelPath' },
+      { title: 'Informal care', path: '/path/2ndLevelPath/3rdLevelPath' },
+      { title: 'Care options', path: null },
+    ],
+  },
+};
+
+export const Example = (args) => <Breadcrumb {...args} />;
+
+export const ExampleInHeader = (args) => {
+  return (
+    <>
+      <style>
+        {`
+          @media only screen and (min-width: 576px) {
+            .breadcrumb-container {
+              margin-right: var(--spacing-xs);
+              margin-left: var(--spacing-xs);
+            }
+          }
+        `}
+      </style>
+      <Header>
+        <Header.UniversalBar primaryLinkText="Helsingin kaupunki" primaryLinkHref="#" />
+        <Header.NavigationMenu>
+          <Header.NavigationLink
+            href="#"
+            label="Link 1"
+            onClick={(event) => event.preventDefault()}
+            active
+            dropdownLinks={[
+              <Header.NavigationLink
+                href="#"
+                label="Test"
+                dropdownDirection={DropdownDirection.Dynamic}
+                active
+                dropdownLinks={[
+                  <Header.NavigationLink href="#" label="Nested" />,
+                  <Header.NavigationLink href="#" label="Nested" />,
+                  <Header.NavigationLink href="#" label="Nested" />,
```

Comment:
Changing these links to match the bread crumb links would be really beneficial to make it clear how these components work together 👍 Then there couldn't be so many breadcrumb links though as navigation menu only supports up to 3rd level nested links. I don't know if it's possible to have deeper breadcrumbs?

Reply:
Matched now. Originally I added long breadcrumb so multiline examples are visible in the Storybook. But for example, hel.fi has 5 level breadcrumbs. All links do not have to be in the menu to be in the breadcrumb.

https://www.hel.fi/en/health-and-social-services/senior-services/informal-care/day-activities-for-the-elderly